### ISSUE TITLE
width of link relative to whole time interval when filtering by time

### ISSUE THREAD

**Issue by csmontt** [FlowmapBlue/flowmap.blue#29]
It seems that the width of a link its determined by the flow between nodes at the time the time filtered is applied and not in relation to the expected flow for that link considering the whole time interval. For example, the attached images show a very similar link width at different time intervals but for very different number of trips, 62 trips vs 8 trips.

I was just wondering if there exists another way of scaling the link's width so they remain comparable over time, or if this featured doesn't exist. 

Finally, many thanks for this tool!

![image](https://user-images.githubusercontent.com/7809239/94204764-c8d49000-feb9-11ea-83db-d9b429db6bce.png)

![image](https://user-images.githubusercontent.com/7809239/94204916-151fd000-feba-11ea-8977-df8410a3624a.png)

**Comment by ilyabo** [comment]
Yes, this is a known issue. I would like to solve it somehow. But it's quite tricky, esp. because of the "Adaptive scales". Every time the viewport is changed I would need to recalculate the extent for every possible position of the selected time range.
